yarn-project/sequencer-client/src/publisher/l1-publisher.ts

```diff
@@ -361,7 +410,23 @@ export class L1Publisher implements L2BlockReceiver {
   private async sendProcessTx(encodedData: L1ProcessArgs): Promise<string | undefined> {
```

Comment:
the doc comments have been lost, as part of the deletion of the interface

Reply:
Can you elaborate a bit more? 

The comments are gone because they were on the interface and you would like them to be added here? I did not move those as I thought the comments were not needed as the functions in here were already undocumented 🤷